lib/money.rb

```diff
@@ -41,9 +41,9 @@ class << self
     def default_currency=(val)
       block =
         if val.respond_to?(:call)
-          -> { Money::Currency.new(val.call) }
```

Comment:
What was the reason for `-> { Money::Currency.new(val.call) }`? You removed it, but I'm just making sure that this wasn't a workaround for some weird use-case.